src/Symfony/Component/Dotenv/Dotenv.php

```diff
@@ -45,10 +45,11 @@
      * @throws FormatException when a file has a syntax error
      * @throws PathException   when a file does not exist or is not readable
      */
-    public function load($path/*, ...$paths*/)
+    public function load($path, ...$paths)
```

Comment:
@Taluu writing `$dotenv->load()` in your code which is always a no-op because you misused it is a bad DX (and this is the experience talking: it was the initial API of the component when merging it in Symfony)